pandas/tests/dtypes/test_dtypes.py

```diff
@@ -115,6 +115,10 @@ def test_construction_from_string(self, dtype):
         with pytest.raises(TypeError, match=msg):
             CategoricalDtype.construct_from_string("foo")
 
+    def test_pyarrow_string_import_error(self, dtype):
+        # GH-44276
```

Comment:
can you flesh this out a tiny bit e.g. 'dont raise if pyarrow is not installed'

Reply:
Done